src/safariservices.cs

```diff
@@ -156,6 +156,7 @@ interface SFSafariPage
 
 	[Mac (10,12)]
 	[Protocol]
+	[Model]
```

Comment:
I don't think we want a `Model` class of this Protocol it is not a `FooDelegate` and it seems it is only used by another class to conform to it. So I would remove `Model` and `BaseType` on this one